### PR TITLE
Globalize

### PR DESCRIPTION
**Pull request by Setogit**
connect to strongloop-internal/scrum-nodeops#1157

### CODE REVIEW

File: lib/upgrade-db.js [hunk 1]
```diff
@@ -16,7 +17,7 @@ function tryUpgradeDb(baseDir, memoryDbPath, dryRun, callback) {
     debug('Ensuring %s doesn\'t exist', meshDbPath);
     if (!err) {
       debug('Error: %s exist', meshDbPath);
-      return callback(new Error('Database ' + meshDbPath + ' already exists'));
+      return callback(new Error(g.t('Database %s already exists', meshDbPath)));
```
Comment (sam-github) [inline lib/upgrade-db.js]: why not `g.Error()`?

File: lib/drivers/common/pack-receiver.js [hunk 2]
```diff
@@ -90,8 +91,8 @@ PackReceiver.prototype.getTarballHash = function(req, res, tarballPath) {
   var self = this;
 
   tarballStream.on('error', function(err) {
-    console.log('Read %s for sha failed: %s', tarballPath, err);
-    self.sendError(res, 'Read %s for sha failed: %s', tarballPath, err);
+    g.log('Read %s for sha failed: %s', tarballPath, err);
+    self.sendError(res, g.t('Read %s for sha failed: %s', tarballPath, err));
```
Comment (sam-github) [inline lib/drivers/common/pack-receiver.js]: `{{sha}}`

File: lib/upgrade-db.js (hunk 1 above)
Comment (Setogit) [inline lib/upgrade-db.js]: done

File: lib/drivers/common/pack-receiver.js (hunk 2 above)
Comment (Setogit) [inline lib/drivers/common/pack-receiver.js]: done

**Comment by Setogit** [comment]
@sl-node test please

**Comment by Setogit** [comment]
@sl-node test please

**Comment by Setogit** [comment]
@slnode test please

**Comment by Setogit** [comment]
This requires https://github.com/strongloop/strong-globalize/pull/5

**Comment by Setogit** [comment]
@sl-node test please

**Comment by sam-github** [comment]
LGTM

but we ***cannot merge*** until strong-globalize has permission to be published to npmjs.org

**Comment by dhmlau** [comment]
Closing as abandoned.

**Pull request closed without merging**